recipes/opsin/1.4.0/opsin.sh

```diff
@@ -42,7 +42,7 @@ for arg in "$@"; do
     esac
 done
 
-if [ "$jvm_mem_opts" == "" ]; then
+if [ "$jvm_mem_opts" == "" ] && [ -z ${_JAVA_OPTIONS+x} ]; then
```

Comment:
Björn -- thanks, nice idea. I haven't done it globally like this but seems like a nice addition to me.